apps/tasks.py

```diff
@@ -649,3 +503,158 @@ def delete_old_objects():
     old_apps = AppInstance.objects.filter(created_on__lt=threshold_time, app__category__name="Develop")
     for app_ in old_apps:
         delete_resource.delay(app_.pk)
+
+
+@shared_task(bind=True, max_retries=None)
+def init_event_listener(self, namespace, label_selector):
+    """
+    The event listener takes the latest event and checks if a corresponding appinstance
+    should be updated. It uses the creation timestamp to always use the status of the youngest pod
+    in a helm release.
+    """
+    api, w = setup_client()
```

Comment:
Could you please use more explicit names? :)
Just `w` is too ambiguous in my opinion.

May be something like this?

```suggestion
    k8s_api, k8s_watcher = setup_client()
```

Reply:
I agree. This syntax is being used in code examples.